sdks/java/core/src/test/java/org/apache/beam/sdk/io/TextIOTest.java

```diff
@@ -520,6 +552,30 @@ public void testWriteDisplayData() {
   }
 
   @Test
+  public void testWriteDisplayDataValidateThenHeader() {
+    TextIO.Write.Bound<?> write = TextIO.Write
+        .to("foo")
+        .withHeader("myHeader");
+
+    DisplayData displayData = DisplayData.from(write);
+
+    assertThat(displayData, hasDisplayItem("fileHeader", "myHeader"));
+    assertThat(displayData, not(hasDisplayItem("validation", false)));
+  }
+
+  @Test
+  public void testWriteDisplayDataValidateThenFooter() {
+    TextIO.Write.Bound<?> write = TextIO.Write
+        .to("foo")
+        .withFooter("myFooter");
+
+    DisplayData displayData = DisplayData.from(write);
+
+    assertThat(displayData, hasDisplayItem("fileFooter", "myFooter"));
+    assertThat(displayData, not(hasDisplayItem("validation", false)));
```

Comment:
This test is for https://issues.apache.org/jira/browse/BEAM-697.  I left a comment about my fix I incorporated into this PR on that Jira, but saw no response.  I can remove both the fix (trivial 2 line fix) and this supporting unit test from this PR if you like.